### ISSUE TITLE
Add version number + start maintanining changelog

### ISSUE THREAD

**Issue by pepsonEL** [tomasbedrich/home-assistant-hikconnect#11]
Maybe good feature will be add changelog and number release.... It will be show it when install in HACS.

**Comment by tomasbedrich** [comment]
Agree! I plan to add it once the initial development is done and it's more stable (no longer beta). Also I want to push it to HACS default so that adding 3rd party repository is no longer needed.

**Comment by pepsonEL** [comment]
Very very good :)
BTW Look again on summary add logo.

**Comment by tomasbedrich** [comment]
Version 1.0.0 is published.

**Comment by pepsonEL** [comment]
Good will be if next version will be update to have example binary sensor which show in HA ring status.
